tvshows/src/theme/foundations/typography/textStyles.ts

```diff
@@ -0,0 +1,52 @@
+const textStyles = {
+  h1: {
+    fontSize: ["54px", "70px"],
```

Comment:
I have a feeling you got the font sizes wrong. The title is 40px with a line height of 52px on web and 23pc with a line height of 32px on mobile. Other sizes also seem to be off.
![image](https://github.com/user-attachments/assets/0678400f-d1a4-428d-a2db-179c840a444f)